docs/guides/deploy.md

```diff
@@ -153,7 +156,7 @@ assert decrypted_result == 49
 
 # Deployment of modules
 
-Deploying a [module](../compilation/composing_functions_with_modules.md) follows the same logic as the deployment of circuits. Assuming a module compiled in the following way:
+Deploying a [module](../compilation/composing_functions_with_modules.md) follows the same logic as the deployment of circuits. For example, a module compiled in the following way:
```

Comment:
"For example, a module compiled in the following way:" has no verb

Reply:
Changed to -> For example,  consider a module compiled in the following way